### DISCUSSION TITLE
Torus & Web3Auth shared Eth Address?

### DISCUSSION THREAD

**Discussion by carlneal** [Web3Auth/Web3Auth#374] [Help]
Hello is Torus wallet the same as web3auth?

Reason Im asking is we are creating wallets with [web3auth.io](http://web3auth.io/) and syncing to Moralis. However we realize people who create an web3auth eth account using our client id, cannot see their same account when going https://app.tor.us/ and login it has a different account number.

The issue is we want to mint nfts and have them view on OpenSea. OpenSea has a Torus connection. But if users login, they have a whole new account id, which doesn't have their nft we minted.

Can you help us here as we are going live in a few days.

**Comment by carlneal** [comment]
It seems it just a checkbox  to ensure compatability with Torus 🙂

https://web3auth.io/docs/developer-dashboard/enable-interoperability

**Comment by YZhenY** [comment]
Web3Auth is not the same as Torus Wallet. 

Web3Auth,  is wallet infrastructure that is plugged into applications or wallets. It results in a standard cryptographic key provider specific to the user and application. While using Web3Auth, you have the flexibility of creating your own wallet within your application or using one of our pre existing adapters, which can handle the complexity of linking your application to a wallet.

Torus Wallet is a wallet that uses Web3Auth - thats why you get different keys. And https://web3auth.io/docs/developer-dashboard/enable-interoperability is one way of showing your Web3Auth key on Torus Walllet.

Read more here too https://web3auth.io/docs/overview/web3auth-and-wallets

**Reply by yashovardhan** [reply]
One thing to add here.

Your user's private key changes for every different Client ID and Verifier combination in Web3Auth. Hence for every different dApp or wallet, the private key will be different, causing the different private key for Torus Wallet as well.
